resources/views/applicant/application_post_03.html.twig

```diff
@@ -26,6 +26,7 @@
     {# Modals #}
 
         {% include "common/modal" with {"page":application} %}
+        {% include "common/modals/skills_need_help" %}
```

Comment:
The skills_need_help modal needs to be included on the `application_post_04` (asset skills) template as well.

Reply:
This was such a small change, I took care of it. cbbb75dcd6f288260fa84c2f4dc229de2706004d